sharedbuffers/mapped_struct.py

```diff
@@ -947,6 +993,24 @@ def _mix_hash(code1, code2):
 @cython.locals(hval=cython.ulonglong, trunc_key=cython.longlong,
     truncated=cython.bint, flt=cython.double, mant=cython.double)
 def _stable_hash(key):
+    """
+    Compute a hash for object ``key`` in a way that is portable across processes and implementations.
+    The computed hash should be appropriate for use in persistent data structures.
+
+    Stable hashing is implemented for:
+
+    * :class:`basestring`
+    * :class:`int`
+    * :class:`long`
+    * :class:`float`
+    * :class:`tuple` and :class:`proxied_tuple`
+    * :class:`frozenset` and :class:`proxied_frozenset`
```

Comment:
Because documentation wasn't generated for them. I'm not sure it's worth generating documentation for them, since their public interface is the tuple/frozenset interface.